examples/allennlp/allennlp_jsonnet.py

```diff
@@ -0,0 +1,59 @@
+"""
+Optuna example that optimizes a classifier configuration for IMDB movie review dataset.
+This script is based on the example of allentune (https://github.com/allenai/allentune).
+
+In this example, we optimize the validation accuracy of
+sentiment classification using AllenNLP cli.
+Since it is too time-consuming to use the entire dataset, we here use a small subset of it.
```

Comment:
If I understand correctly...
```suggestion
Since it is too time-consuming to use the training dataset, we here use a validation dataset instead..
```

Reply:
You're right, thank you.